src/main/java/FlightsReporter.groovy

```diff
@@ -0,0 +1,257 @@
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+
+class FlightsReporter {
+    final static int[] CAPACITIES = new int[] {80, 120, 150, 180, 240}
+
+    static List<Flight> getAllFlightsTo(List<Flight> flights, Airports destination) {
+        flights.findAll { (it.destination == destination) }
+    }
+
+    static List<Flight> getAllFlightsFrom(List<Flight> flights, Airports from) {
+        flights.findAll {it.from == from }
+    }
+
+    static void printFlights(List<Flight> flights) {
+        flights.each {println(it)}
+    }
+
+    static int passengerAmount(List<Flight> flights) {
```

Comment:
Amount is for non-countable things :) Passengers are clearly countable 
Also use a verb please